46sshd/sshd_config

```diff
@@ -7,6 +7,7 @@ GSSAPIAuthentication no
 GSSAPICleanupCredentials no
 UsePAM no
 X11Forwarding no
+AuthenticationMethods publickey
```

Comment:
Makes sense.

Reply:
I've integrated this in https://github.com/gsauthof/dracut-sshd/commit/12a0d4105c171179edae323d7cc9cd65643f9990